phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
I've only spent the last week digging in to all of these issues (so my opinion is _FAR_ from that of an expert), but for my $0.02 if the fluent files are stored in `/var/log` then we probably need to ensure the mounting of that folder, rather than only `/var/log/containers` to ensure the proper sharing of them.